scripts/fhir-works-deploy.sh

```diff
@@ -36,19 +29,15 @@ done
 
 printf "\nFor deployment package, remove old pregenerated files, add all other packages as dependencies, install all dependencies again"
 cd fhir-works-on-aws-deployment
-rm -rf node_modules build/* dist/* 
+rm -rf node_modules build/* dist/*  .yalc
 yalc add fhir-works-on-aws-interface
 for i in "${packages[@]}"
 do
   yalc add $i
 done
+sed -i.bak 's#file:.yalc#./.yalc#g' package.json && rm package.json.bak
```

Comment:
This method will allow the script to work on linux and macOS
https://stackoverflow.com/a/44864004/14310364